### PR TITLE
INT: Flip operator of binary expression

### LINKED ISSUE #2343: Flip '+' itention
Here's this intention for Kotlin:

image

Would be great to have it in Rust!

Take a look at [this PR](https://github.com/intellij-rust/intellij-rust/pull/2208/files) to learn what is needed for an intention:

- [ ] intention itself
- [ ] line in plugin.xml
- [ ] description in resources
- [ ] a bunch of tests

I *think* a relevant intention for Java is here:

https://github.com/JetBrains/intellij-community/blob/3b55cc70faf7346e324637d264802caa337505d2/plugins/IntentionPowerPak/src/com/siyeh/ipp/expression/FlipExpressionIntention.java

### PR DESCRIPTION
**Pull request by t-kameyama**
Closes #2343

### CODE REVIEW

File: src/main/resources/intentionDescriptions/FlipBinaryExpressionIntention/description.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+This intention flips the operator of a binary expression.
```
Comment (Undin) [inline src/main/resources/intentionDescriptions/FlipBinaryExpressionIntention/description.html]: Looks like the description for similar java intention is slightly better (see https://github.com/JetBrains/intellij-community/blob/master/plugins/IntentionPowerPak/src/intentionDescriptions/FlipExpressionIntention/description.html). Maybe use the same description?

**Comment by matklad** [comment]
>What approach do we want to use? @matklad

Java's one seems more useful!

**Comment by t-kameyama** [comment]
Fixed

**Comment by Undin** [comment]
Looks good!
Thanks!
bors r+

**Pull request merged**